monitoring/README.md

```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
 * `monitoring/user-values-prom-operator.yaml`
 * `monitoring/user-values-pushgateway.yaml`
 
+You can modify two types of values to customize your deployment:
+- Environment variables: These variables contain uppercase characters and underscores (for example, `LOG_COLOR_ENABLE=`), and can be specified in the `user.env` file or on the command line. Specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments.
```

Comment:
Environment variables do NOT have to use uppercase and underscores...they often do but that is not a requirement.  I do NOT know if every env variable we support follows that convention.  I'm not sure Bryan meant for that to get into the doc...thought it was just internal guidance to help differentiate between env vars and Helm chart parameters.  If it goes in the doc, we need to weaken the statement (perhaps: _These variables often (or generally?) contain uppercase and underscores_)

Reply:
Removed information about case and characters in variables